src/app/services/websocket-status.service.ts

```diff
@@ -19,6 +19,7 @@ export class WebSocketStatusService {
   }
 
   readonly isReconnectAllowed$ = new BehaviorSubject<boolean>(false);
+  readonly isFailoverRestart$ = new BehaviorSubject<boolean>(false);
```

Comment:
Should be private

Reply:
`isFailoverRestart$` is used in the signin component.